Microsoft.Toolkit.Win32/Microsoft.Toolkit.Forms.UI.Controls/InkCanvas/InkCanvas.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT;
+
+namespace Microsoft.Toolkit.Forms.UI.Controls
+{
+    public class InkCanvas : WindowsXamlHostBaseExt
+    {
+        internal Windows.UI.Xaml.Controls.InkCanvas UwpControl => XamlElement as Windows.UI.Xaml.Controls.InkCanvas;
```

Comment:
`private`

Reply:
Toolbar accesses this as a member when setting its own member property TargetInkCanvas